lib/nexmo_developer/app/helpers/static_helper.rb

```diff
@@ -0,0 +1,14 @@
+module StaticHelper
```

Comment:
We don't want to add another layer of abstraction here - would you mind remove this helper file please?